### PR TITLE
Adding model eval hw

### PR DESCRIPTION
**Pull request by DeronDMV**

### CODE REVIEW

File: deron/Homework/Class_10_Homework.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+from sklearn.linear_model import LogisticRegression
+from sklearn.cross_validation import train_test_split, cross_val_score
+
+# Read in Data
+features = ['ID', 'RI', 'Sodium', 'Mg', 'Al', 'Si', 'K', 'Ca', 'Ba', 'Fe', 'type_of_glass']
+glass_data_url = 'http://archive.ics.uci.edu/ml/machine-learning-databases/glass/glass.data'
+glass_data = pd.read_table(glass_data_url, sep=',', names=features, index_col='ID')
+
+# Add 'Binary' to data
+glass_data['binary'] = glass_data.type_of_glass.map({1:0, 2:0, 3:0, 4:0, 5:1, 6:1, 7:1})
```
Comment (sinanuozdemir) [inline deron/Homework/Class_10_Homework.py]: you could also use `glass_data['binary'] = np.where(glass_data.type_of_glass < 5, 0, 1)`

File: deron/Homework/Class_10_Homework.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+# Print Confusion Matrix
+print(ConfusionMatrix(list(y_test), list(preds)))
+
+# print Accuracy
+print metrics.accuracy_score(y_test, preds)
+
+# Compare Scores of logReg, KNN=1, and KNN=3
+scores = cross_val_score(logreg, X, y, cv=10, scoring='mean_squared_error')
+np.mean(np.sqrt(-scores))
```
Comment (sinanuozdemir) [inline deron/Homework/Class_10_Homework.py]: you could also do `cross_val_score(...).mean()`

File: deron/Homework/Class_10_Homework.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+from sklearn.neighbors import KNeighborsClassifier
+knn = KNeighborsClassifier(n_neighbors=1)
+scores_KNN_One = cross_val_score(knn, X, y, cv=10, scoring='mean_squared_error')
+np.mean(np.sqrt(-scores_KNN_One))
+
+from sklearn.neighbors import KNeighborsClassifier
+knn2 = KNeighborsClassifier(n_neighbors=3)
+scores_KNN_Two = cross_val_score(knn2, X, y, cv=10, scoring='mean_squared_error')
+np.mean(np.sqrt(-scores_KNN_Two))
```
Comment (sinanuozdemir) [inline deron/Homework/Class_10_Homework.py]: What can you conclude from running these three tests! Which ones preformed better ? :)

**Comment by sinanuozdemir** [comment]
Good job! Let us know if you have any particular questions on the homework.

File: deron/Homework/Class_10_Homework.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+preds = logreg.predict(X_test) 
+
+# Print Confusion Matrix
+print(ConfusionMatrix(list(y_test), list(preds)))
+
+# print Accuracy
+print metrics.accuracy_score(y_test, preds)
+
+# Compare Scores of logReg, KNN=1, and KNN=3
+scores = cross_val_score(logreg, X, y, cv=10, scoring='mean_squared_error')
```
Comment (justmarkham) [inline deron/Homework/Class_10_Homework.py]: @DeronDMV Are you sure you want to use mean_squared_error? Think about what type of problem this is, and whether or not that scoring metric (aka "error metric") makes sense for this problem. Let us know if you have questions!
